### ISSUE TITLE
Plugin deployment doc incorrectly claims that built-in napari plugin installer doesn't install from conda 

### ISSUE THREAD

**Issue by TimMonko** [napari/docs#532]
## 🐛 Bug

[Deploying plugin docs](https://napari.org/stable/plugins/testing_and_publishing/deploy.html) says that the napari plugin installer doesn't currently install from conda... but I'm pretty sure it does? 

I broke a [conda-forge-feedstock](https://github.com/conda-forge/napari-ndev-feedstock) many versions ago, and never got around to figuring out how to make conda distributions actually work. Because of this note, I just assumed that conda didn't work in napari, but then I had a user report that my plugin looks way different than expected -- no surprise when it was nearly a year out of date in conda, and I naively just assumed pip was how people got the plugin. I think the installer defaults to conda now.

 I'll try to improve the [Deploying to Anaconda](https://napari.org/stable/plugins/virtual_environment_docs/2-deploying-your-plugin.html#deploying-to-anaconda) section as I figure things out. So that people like me don't give up like I did.

**Comment by Czaki** [comment]
You are right. The old bundle version was pip only, and we still have not updated part of docs.

**Comment by melissawm** [comment]
Updating the plugins docs is my priority for the next couple of weeks. There are several issues about this that I want to address and this is one of them. Thanks for the report!

**Comment by TimMonko** [comment]
Feel free to ping me to look things over. If I find some time, you can also let me any documentation I could work on. Right now I'm working back through thinking about my napari plugin since I've learned much in the past 2 years that I had originally just brute forced. (which I think was, at least, a good thing for getting users like me started!)
